src/Servers/IIS/AspNetCoreModuleV2/Symbols/Microsoft.AspNetCore.ANCMSymbols.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <Import Project="..\..\build\assets.props" />
+
+  <ItemGroup Condition="'$(BuildIisNativeProjects)' == 'true' AND '$(VCTargetsPath)' != ''">
+    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj" Platform="x64" />
+    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj" Platform="Win32" />
+    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj" Platform="x64" />
+    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj" Platform="Win32" />
+  </ItemGroup>
+
+  <Target Name="AddPackNativeComponents" BeforeTargets="_GetPackageFiles" Condition="$(PackNativeAssets) == 'true'">
+    <ItemGroup>
+      <None Include="%(ShimComponents.DllLocation)"
+            Pack="true"
+            PackagePath="runtimes/win-%(ShimComponents.Platform)/native/$(NativeAssetsTargetFramework)" />
+      <None Include="%(ShimComponents.PdbLocation)"
+            Pack="true"
+            PackagePath="runtimes/win-%(ShimComponents.Platform)/native/$(NativeAssetsTargetFramework)" />
```

Comment:
Wouldn't this mean it ends up in `runtimes/win-Win32/native/...`? It should be `win-x86` for 32-bit and `win-x64` for 64-bit. I know that these are mostly bogus packages, but I really think we should align to the conventions anyway, especially if Arcade is going to consume these at some point. We don't know what restrictions they may put on it and while we can always have input, it's better to just *be* a correct NuGet package to start with :).

If I'm wrong about the folder path, then there's no problem :)

Reply:
No, it's win-x86.

Reply:
🎉